src/main.c

```diff
@@ -317,7 +317,7 @@ void handleSign(uint8_t p1, uint8_t p2, const uint8_t *workBuffer, uint16_t data
       dataLength--;
     }
     else {
-      txContext.txType = CELO_LEGACY;
+      THROW(0x6501);
```

Comment:
It would be better to use a variable there i.e. THROW(STATUS_CODE_DEPRECATED)

Reply:
Where do you suggest I declare it ? In `globls.h`? 

Also there is no use of such a pattern in the code, if you look at any time it throws a status code (error or not) it uses the code directly instead of a variable. So should i change them all, so they all use variables, or let this one not use a variable ? what do you think ? 

P.S.: The file  `/doc/ethapp.asc` has a list all the status code and their meanings

Reply:
If there is no such structure already in place let's leave it line that. It would be great if you could replace the raw status codes to global constants in a future PR.
I would suggest you declare those in a new file with all other constant APDU declarations (i.e. [apdu_constants.h](https://github.com/LedgerHQ/app-ethereum/blob/develop/src/apdu_constants.h#L38) in app-eth)
It would improve code readability and we won’t have to go back and forth from the code to the docs